nfts/src/tests/extrinsics.rs

```diff
@@ -9,375 +9,548 @@ fn origin(account: u64) -> mock::Origin {
 	RawOrigin::Signed(account).into()
 }
 
-/* fn root() -> mock::Origin {
+fn root() -> mock::Origin {
 	RawOrigin::Root.into()
-} */
+}
 
-mod transfer {
+mod create {
 	use super::*;
 
 	#[test]
-	fn cannot_transfer_delegated_nfts() {
-		ExtBuilder::new_build(vec![(ALICE, 100)]).execute_with(|| {
-			let nft_id = NFTs::create_nft(ALICE, vec![0], None).unwrap();
-			assert_ok!(NFTs::set_viewer(nft_id, Some(BOB)));
+	fn create() {
+		ExtBuilder::new_build(vec![(ALICE, 1000)]).execute_with(|| {
+			let alice: mock::Origin = origin(ALICE);
+			let data = NFTData::new_default(ALICE, vec![1], vec![50]);
+			let alice_balance = Balances::free_balance(ALICE);
 
-			let ok = NFTs::transfer(origin(ALICE), nft_id, BOB);
-			assert_noop!(ok, Error::<Test>::CannotTransferDelegatedNFTs);
+			// Create NFT with new serie id while there is no series already registered
+			let ok = NFTs::create(
+				alice.clone(),
+				data.ipfs_reference.clone(),
+				Some(data.series_id.clone()),
+			);
+			assert_ok!(ok);
+			let nft_id = NFTs::nft_id_generator() - 1;
+
+			// Final state checks
+			assert_eq!(NFTs::series_id_generator(), 0);
+			let nft_series_details = Some(NFTSeriesDetails::new(ALICE, true));
```

Comment:
Put it above all the assert_eq (It should be before line 36)

Reply:
Done